collections/_drafts/agent-forwarding.md

```diff
@@ -0,0 +1,297 @@
+`gpgconf --launch gpg-agent` to explicitly start it without first having to use
+a GnuPG command.
+
+`gpg-agent` is a daemon to manage secret (private) keys independently from any
+protocol. It is a backend for gpg, gpgsm, and other utilities.
+
+GPG Agent Configuration
+
+There are a few configuration files needed for the operation of the agent. They
+may all be found in the current GnuPG home directory, which defaults
+to ~/.gnupg.
+
+[[windows]]
+
+However, there may be problems on Windows systems with Gpg4Win installed;
+**Gpg4Win may have changed this default GnuPG home directory location** to an
+AppData subdirectory (i.e., C:\\Users\\<username>\\AppData\\Roaming\\gnupg).
+This location, however, is not the location our GPG agent will be using, so to
+reaffirm our preference for the default location, we will set
+the GNUPGHOME environment variable to ~/.gnupg in the Git Bash startup script by
+running the following.
+
+echo 'export GNUPGHOME="~/.gnupg"' >> .bashrc
+
+[[endwindows]]
+
+Setting environment variables
+
+Add the following lines to your .bashrc or whatever initialization file is used
+for all shell invocations:
+
+GPG_TTY=$(tty)
+
+export GPG_TTY
+
+This variable may only be helpful if you use `pinentry\-curses` (the
+terminal-based pin entry program).
+
+[[windows]]
+
+On Windows systems, you should add the above lines to the ~/.bashrc file for use
+by Git Bash.
+
+[[endwindows]]
+
+GnuPG configuration
+
+It's important to note that GnuPG on the remote system still needs your public
+GPG keys to work correctly. So you have to ensure they are available on the
+remote system even if your secret keys are not.
+
+:::note{.note}
+
+If you use VSCode with the remote extension pack, you may skip this step and
+move on to the next section. This step of copying over your local public GPG
+keyring into the remote container gets done automatically by the extension.
+
+:::
+
+:::excerpt{.quote} <!-- TODO(DerekNonGeneric): link to source of below -->
+
+<abbr title="Secure Copy Protocol">[SCP]{#scp .dfn}</abbr> is a means of
+securely transferring [computer files][] between a local [host][] and a remote
+host or between two remote hosts. It is based on the [Secure Shell][] (SSH)
+protocol.[^1] "SCP" commonly refers to both the Secure Copy Protocol and the
+program itself.[^2]
+
+<!-- TODO(DerekNonGeneric): include how SCP's not:
+- preferable over using SFTP for remote file transfer
+- a bad choice for quick local transfers btwx host and guest vm/container(s)
+-->
+
+:::
+
+<!-- LINK LABEL DEFINITIONS: START -->
+
+[computer files]: https://en.wikipedia.org/wiki/Computer_file
+[host]: https://en.wikipedia.org/wiki/Server_(computing)
+[Secure Shell]: https://en.wikipedia.org/wiki/Secure_Shell
+
+[^1]: https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-1
+[^2]: https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-Pechanec-2)
+
+<!-- LINK LABEL DEFINITIONS: END -->
```

Comment:
```suggestion
<!-- LINK LABEL DEFINITIONS - START -->

[computer files]: https://en.wikipedia.org/wiki/Computer_file
[host]: https://en.wikipedia.org/wiki/Server_(computing)
[Secure Shell]: https://en.wikipedia.org/wiki/Secure_Shell

[^1]: https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-1
[^2]: https://en.wikipedia.org/wiki/Secure_copy_protocol#cite_note-Pechanec-2)

<!-- LINK LABEL DEFINITIONS - END -->
```

☝️ after re-checking notes, have noticed a few inconsistencies here and elsewhere regarding the way blocked-off links are delimited w/ a comment&hellip;

it is a dash (one dash and Prettier docs are where we saw that initially); not a convention we have been good at following as most needs to be corrected, but going forward, let's not deviate from Prettier docs code examples lol (seriously, we might need to write a lint rule)

🔗 https://prettier.io/docs/en/ignore#range-ignore